packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,30 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [LoopBack CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
```

Comment:
> To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed

I don't think this line is needed at all. The user had to have node and npm installed in order to install loopback 4 to begin with, and this readme will appear after they've run the CLI to create a project.

Actually I think the entire `### Installation` should be removed as first step.

Once the CLI for `lb4 app` has completed, certain dependencies have already been added to package.json, and they have been installed .

The first step can be to `### Run` ,  since it is a fully runnable application at this point with `/ping` endpoint.

Reply:
I totally agree. But the moment this is uploaded to Git and someone new arrives, they will have to know what to do

Reply:
> But the moment this is uploaded to Git and someone new arrives, they will have to know what to do

Apologies, but I do not understand this line of thought. Yes this readme will be stored in git as part of cli scaffolding. But a user doesn't visit this file to find out how to use lb4. They go to our documentation website. This readme gets created after : 1) someone installs node and npm, 2) installs loopback 4 cli, and 3) after they run `lb4 app`. A new user will likely be following either: 1) https://loopback.io/doc/en/lb4/Getting-started.html or 2) https://loopback.io/doc/en/lb4/todo-tutorial.html or 3) https://loopback.io/doc/en/lb4/Command-line-interface.html.

Reply:
I understand. I am going to delete the "installation" section